ranger-tools/src/main/python/stress/stress-hbase-loadgenerator.py

```diff
@@ -0,0 +1,106 @@
+import subprocess
+import time
+import argparse
+import os
+from datetime import datetime
+
+def increase_memory_for_loadgenerator():
+    try:
+        cmd = "export HBASE_OPTS='-Xmx10g'"
+        print(cmd)
+        op = subprocess.call(cmd, shell=True)
+        print("Output:", op)
+    except subprocess.CalledProcessError as e:
+        print("Error in setting HBASE_HEAPSIZE:", e)
+        exit(1)
+def login():
+    try:
+        cmd = "kinit -kt <systest.keytab> systest"
```

Comment:
Instead of hardcoding the value it is better to provide the keytab as an argument. Alternatively, doing the kinit externally and documenting the steps in readme also works.